src/midiEditorCommands.cpp

```diff
@@ -500,4 +500,23 @@ void cmdFocusNearestMidiEvent(Command* command) {
 		}
 	}
 }
+
+static BOOL CALLBACK cbFindFilterWindow(HWND hWnd, LPARAM lParam) {
+	WCHAR szText[128];
+	if ((GetWindowTextW(hWnd, szText, 128) != 0) &&
+		!wcsncmp(szText, L"Filter Events", 13)) {
+		*((HWND *)lParam) = hWnd;
+		return false;
+	}
+	return true;
+}
+
+void cmdMidiFilterWindow(Command *command) {
+	HWND editor = MIDIEditor_GetActive(), filter = NULL;
+	MIDIEditor_OnCommand(editor, command->gaccel.accel.cmd);
+	// TODO: we could also check the command state was "off", to skip searching otherwise
+	EnumWindows(cbFindFilterWindow, (LPARAM)&filter);
```

Comment:
I think you can simplify (and get rid of the callback) to this:

```
HWND filter = FindWindow(WC_DIALOG, L"Filter Events");
```

Reply:
Original idea was to make it correct way. There can be several MIDI editors open in REAPER, and so several open "Filter Events" dialogs. But I have not found a good way to detect which dialog is for which editor. And in this case you are right, FindWindow produce the same result without any extra code.